server/szurubooru/func/user_tokens.py

```diff
@@ -0,0 +1,152 @@
+import pytz
+from datetime import datetime
+from pyrfc3339 import parser as rfc3339_parser
+from typing import Any, Optional, List, Dict, Callable
+from szurubooru import db, model, rest, errors
+from szurubooru.func import auth, serialization, users, util
+
+
+class InvalidEnabledError(errors.ValidationError):
```

Comment:
This exception doesn't seem to have tests.

Reply:
Hmm, that exception shouldn't be there. enabled can't be None because it is coming from ctx.get_param_as_bool().

I've removed the exception and also the guard code in the update method.